golonomic.go

```diff
@@ -18,36 +18,52 @@ const (
 
 var inverse = mat.NewDense(3, 3, nil)
```

Comment:
maybe you can be more specific with the naming, what is the matrix actually used for?

Reply:
I doubt I can capture that in a variable name so I added a description